tests/event_finder/test_event_finder.py

```diff
@@ -38,7 +38,7 @@ def test_get_window_list(self):
                                       bandwidth=None
                                   )
                                   )
-        overhead_windows = EventFinderRhodesMill(list_of_satellites=list_of_satellites_with_frequency, reservation=reservation, azimuth_altitude_path=None).get_overhead_windows()
+        overhead_windows = EventFinderRhodesMill(list_of_satellites=list_of_satellites_with_frequency, reservation=reservation, azimuth_altitude_path=None, search_time_start=reservation.time.begin, search_time_end=reservation.time.end).get_overhead_windows()
```

Comment:
`search_time_start=reservation.time.begin, search_time_end=reservation.time.end` should be TimeWindow?

Reply:
resolved